README.md

```diff
@@ -0,0 +1,253 @@
+        langDir: "lang/",
+        defaultLocale: "en",
+        vueI18n: {
+          silentTranslationWarn: true,
+          silentFallbackWarn: true
+        }
+      }
+    ],
+    [
+      "vue-toastification/nuxt",
+      {
+        transition: "Vue-Toastification__fade",
+        maxToasts: 20,
+        newestOnTop: true
+      }
+    ]
+  ]
+}
+```
+
+To use module with nuxt `publicRuntimeConfig` use the below:
+
+````js
+modules: [
+    '@geeks.solutions/nuxt-sections',
+    ...
+],
+
+publicRuntimeConfig: {
+    sections: {
+      projectId: '62ff7827628cfa00099de9e1',
+      projectUrl: 'http://localhost:3000',
+      environment: 'testing',
+      queryStringSupport: "enabled"
+    }
+}
+````
+
+3. Add the sections component on the page(s) of your choice
+
+````vue
+<template>
+  <Sections
+    :admin="admin"
+    :page-name="pageName"
+    :lang="lang"
+    :variations="[]"
+  />
+</template>
+
+<script>
+export default {
+  name: 'IndexPage',
+  data() {
+    return {
+      pageName: "Home"
+    };
+  },
+  computed: {
+    lang() {
+      return this.$i18n.locale.toString()
+    },
+    admin() {
+      return !!this.$cookies.get("sections-auth-token")
+    }
+  }
+}
+</script>
+
+````
+
+Here we load props from data on the page, the admin prop is used to indicate if the edit interface for the page should display or not. Sections uses a cookie named `sections-auth-token` to store a user token to secure communications for page editing actions, if the cookie is found it assumes the edit button should show. In case you want to know whether the sections are loaded, You can use @finishLoad in your Sections component with a method name. Don't forget to add the method in the methods object.
+
+To get the UserToken and have it stored in the above cookie, simply set sections on a page of your website, then head to your Sections project page (where you have your projectID) and set the login redirect url properly, then hit connect sections to my app. This library will receive and process the request to generate the UserToken and store it in the cookie for further use. for more information check the [docs](https://sections.geeks.solutions/docs/frontend/index.html)
+
+If you now want to move on and start providing local and static sections for your website editor, or customize the display of dynamic or configurable ones, read below.
+
+# How it works
+
+Sections server comes with a Wysiwyg and any public sections out of the box.
+The Wysiwg can be added to any page and will display of the box thanks to internal components, yet if you wish to override anything (the view, the icon or the form) you can do so be redeclaring any (or all) component(s) in your project.
+
+You can also define your own local and static section types and you have the ability to control the way any section will display on your website.
+
+You do this by creating custom VueJS components and placing them in the right configurations folder.
```

Comment:
It is still relevant because the forms / views and type-icons are still used with the same structure as the ones in Vue library so inside sections folder > views / forms and type-icons

Reply:
I know, I was asking about the difference between a vue project and a nuxt project

Reply:
you haven't replied to this comment

Reply:
There is no difference, it is exactly the same